selfdrive/car/hyundai/hyundaicanfd.py

```diff
@@ -191,6 +183,20 @@ def create_adrv_messages(packer, CAN, frame):
       'SET_ME_9': 0x9,
     }
     ret.append(packer.make_can_msg("ADRV_0x160", CAN.ECAN, values))
+  return ret
+
+
+def create_adrv_messages(packer, CAN, frame):
+  # messages needed to car happy after disabling
```

Comment:
I realize this is just a moved comment but "needed to car happy" is missing a verb or something.